topic_folders/communications/guides/submit-job-posting.md

```diff
@@ -0,0 +1,41 @@
+## Sharing Job Postings
+
+The Carpentries has two main mechanisms by which to share community job postings: One is within our newsletter, [Carpentries Clippings](https://carpentries.org/newsletter/) and the other is our [#jobs channel on Slack](https://join.slack.com/share/zt-jw6ndhkl-rKyqtKGiBZnVtojOUDtgLw)
+
+### Criteria for Posting
+In order to ensure that job postings featured on Carpentries forums are applicable and of relevance to our community, The Carpentries Core Team has a list of criteria that a posting must meet in order to be shared. If the posting does not meet the criteria, then the post will be removed or modified after discussion with the poster.
+
+#### Here are the criteria we consider when deciding whether to feature or approve a job posting:
+- **Organisational Alignment with The Carpentries**
+  - Does the organisation which is hiring have similar goals to The Carpentries?
+  - Do our goals directly support each other?
+- **Carpentries Code of Conduct**
+  - Does the organisation abide by our Code of Conduct and/or have their own published Code of Conduct that has the same priorities as The Carpentries'
+  - Note: If The Carpentries publishes a posting from an organisation and is then made aware by our community of incompatibility with our Code of Conduct, we will remove the posting
+- **Quality of The Posting**
+  - Does the posting bear relevance to our community?
+  - Are the skills and experiences sought by the posting organisation compatible with the skills our community has or seeks to gain?
+  - Does the posting mention The Carpentries by name?
+- **Prior Knowledge/work with the Posting Organisation**
+  - Often times, The Carpentries posts job opportunities from affiliate or member organisations.
+  - If we have prior interactions and knowledge of the organisation, this is a strong indicator of a valuable opportunity post that our community will appreciate.
+
+Note: These criteria are not strictly quantitative and rely on the judgment of the core team.
+
+### Sharing Postings on Carpentries Clippings
+Carpentries Clippings is a biweekly newsletter to members of our community featuring community news, program updates of note, and opportunities relevant to the wider Carpentries community.
+
+In order to share a job posting that we will feature in Carpentries Clippings, please follow the [Sharing Relevant Announcements](https://docs.carpentries.org/topic_folders/communications/guides/submit-news-item.html) guide.
+
+### Sharing Job Postings on Slack
+Once you have read the criteria above and joined the #jobs channel on The Carpentries Slack, you can share relevant announcements to the group. Please include the following information when posting:
+
+- Job Title
+- Organisation
+- Location (if applicable)
+- Posting/Application Deadline
+- Link for more information and to apply
```

Comment:
I think it would be good to specifically state that saying that more information is available by DM is not acceptable.